### DISCUSSION TITLE
Fontawesome instead of ionicons

### DISCUSSION THREAD

**Discussion by Loukili-Rachid** [TiagoSilvaPereira/vemto-issues#270] [Ideas & Suggestions]
i suggest Fontawesome instead of ionicons as a default icons

i think Fontawesome has more icons then ionicons..

**Comment by TiagoSilvaPereira** [comment]
Hi @Loukili-Rachid it would be cool, but not currently possible, as font awesome needs a singular key for integration, and you receive the CDN link in your email inbox. Furthermore, FontAwesome is not totally free.

I recommend trying to create a Vemto plugin in this case, so you can replace the ionicons with fontawesome when necessary
